internal/state/relationship/capturer_test.go

```diff
@@ -313,23 +314,135 @@ var _ = Describe("Capturer", func() {
 				})
 			})
 		})
-		Describe("Edge cases", func() {
-			BeforeEach(func() {
-				capturer = relationship.NewCapturerImpl()
+	})
+	Describe("Capture namespace and gateway relationships", func() {
+		var gw *v1beta1.Gateway
+		var nsNoLabels, ns *v1.Namespace
+
+		BeforeEach(OncePerOrdered, func() {
```

Comment:
I don't think you want `OncePerOrdered` here. The test container (Describe) is not `Ordered` and the test cases that follow assume an unordered container.

Reply:
Ah, copy/paste error.